src/vivarium/framework/randomness/index_map.py

```diff
@@ -12,11 +12,12 @@
 
 from __future__ import annotations
 
-import datetime
-from typing import Any, Optional, Union
+from datetime import datetime
+from typing import Any
 
 import numpy as np
 import pandas as pd
+from pandas.api import types as pdtypes
```

Comment:
I've seen that `numpy.typing` is often imported as `import numpy.typing as npt`. Is there a similar convention for this? If not, do we prefer `pdtypes` or `pd_types`?

Reply:
Oh i prefer pdt then